tests/TestCase/View/XmlViewTest.php

```diff
@@ -307,7 +306,8 @@ public function testRenderSerializeWithFalseyValue(): void
 
         $expected = Xml::build(['response' => $data])->asXML();
         $this->assertSame($expected, $result);
-        $this->assertStringContainsString('<testing>0</testing>', $result);
+        $expectedString = $expectedValue === null ? '<testing/>' : '<testing>' . $expectedValue . '</testing>';
```

Comment:
Why can't `$expectedValue` have the value of `$expectedString` directly?
The values ​​of the first 3 cases would be the same, but it would all become much more readable and without the need for additional steps.

Reply:
```php
#[TestWith([0, '<testing>0</testing>'])]
#[TestWith(['0', '<testing>0</testing>'])]
#[TestWith([false, '<testing>0</testing>'])]
#[TestWith(['', '<testing/>'])]
```